package.json

```diff
@@ -37,10 +37,12 @@
   "devDependencies": {
     "@babel/core": "7.1.6",
     "@babel/helper-plugin-test-runner": "7.1.0",
+    "@babel/parser": "^7.2.0",
     "@babel/plugin-proposal-class-properties": "7.1.0",
     "@babel/plugin-transform-runtime": "7.1.0",
     "@babel/preset-env": "7.1.6",
     "@babel/runtime-corejs2": "7.1.5",
+    "@babel/traverse": "^7.1.6",
```

Comment:
Let's not use the hat version here, can we specify the exact pinned version (7.1.6).

Reply:
done